content/techniques/queues.md

```diff
@@ -194,6 +194,27 @@ const job = await this.audioQueue.add(
 );
 ```
 
+#### Providers
+
+Whether the processor needs to depend on a request scoped provider or be explicitly registered as a scoped processor, as follows:
+
+```typescript
+@Processor({
+  name: 'audio',
+  scope: Scope.REQUEST,
+})
+```
+
+then you can inject the `JOB_REF` via the constructor.
+
+```typescript
+constructor(@Inject(JOB_REF) jobRef: Job) {
+   console.log(jobRef);
+}
+```
+
+> info **Hint** to import JOB_REF: ``` import { JOB_REF } from '@nestjs/bull' ```
```

Comment:
This is possible, but you'll need to escape the `{` and `}`. It may be cleaner and easier to do

```suggestion
> info **Hint** The `JOB_REF` is imported from `@nestjs/bull`
```

Otherwise, to escape the brackets you need to do `{{ '{' }}` and `{{ '}' }}`